ambry-store/src/main/java/com.github.ambry.store/Log.java

```diff
@@ -127,6 +127,22 @@ public int appendFrom(ByteBuffer buffer) throws IOException {
     return activeSegment.appendFrom(buffer);
   }
 
+  /**
+   * {@inheritDoc}
```

Comment:
don't think you need this - function not inheriting from anything?

Reply:
Fixed.